sdk/core/azure-core/src/main/java/com/azure/core/util/Context.java

```diff
@@ -23,6 +24,9 @@
 public class Context {
     private final ClientLogger logger = new ClientLogger(Context.class);
 
+    /**Key used to override headers in HttpRequest. The Value for this key should be {@link HttpHeaders}.*/
+    public static final String AZURE_REQUEST_HTTP_HEADERS_KEY = "azure-http-headers-key";
```

Comment:
This isn't the right place for this field. It feels like it should be in the policy, not here.